packages/manager/apps/sap-features-hub/src/pages/installation/stepSystemInformation/installationStepSystemInformation.constants.tsx

```diff
@@ -1,42 +1,42 @@
 import { SystemForm, TextInputData } from '@/types/form.type';
+import { FORM_LABELS } from '@/constants/form.constants';
 
 // Un SID SAP comporte 3 caractères alphanumériques en majuscule et commence par une lettre.
```

Comment:
I known is not this PR put this comment should be in english if we this rest in code